doc/Pull Request Acceptance Checklist.md

```diff
@@ -0,0 +1,15 @@
+# Pull Request Acceptance Checklist
+
+## Beagle Project
+ * detailed expanatory Javadoc
+ * No checkstyle violations (errors or warnings)
+ * 100% Code coverage by JUnit Tests
+ * JUnit test check were created by another person and are complete (check everything possible)
```

Comment:
> (check everything possible)

well that’s impossible and quite vague. Can we thing of a more precise definition of when a PR is sufficiently tested? Is that even possible?